packages/fuselage-ui-kit/src/blocks/VideoConferenceBlock/VideoConferenceBlock.tsx

```diff
@@ -0,0 +1,211 @@
+  const surfaceType = useSurfaceType();
+
+  const { action, viewId, rid } = useContext(kitContext);
+
+  if (surfaceType !== 'message') {
+    return <></>;
+  }
+
+  if (!callId || !rid) {
+    return <></>;
+  }
+
+  const getUserAvatarPath = useUserAvatarPath();
+  const result = useVideoConfDataStream({ rid, callId });
+
+  const joinHandler: MouseEventHandler<HTMLButtonElement> = (e): void => {
+    action(
+      {
+        blockId: block.blockId || '',
+        appId,
+        actionId: 'join',
+        value: block.blockId || '',
+        viewId,
+      },
+      e
+    );
+  };
+
+  const callAgainHandler: MouseEventHandler<HTMLButtonElement> = (e): void => {
+    action(
+      {
+        blockId: rid || '',
+        appId,
+        actionId: 'callBack',
+        value: rid || '',
+        viewId,
+      },
+      e
+    );
+  };
+
+  const openCallInfo: MouseEventHandler<HTMLButtonElement> = (e) => {
+    action(
+      {
+        blockId: callId,
+        appId,
+        actionId: 'info',
+        value: rid,
+        viewId,
+      },
+      e
+    );
+  };
+
+  if (result.isSuccess) {
+    const { data } = result;
+
+    if ('endedAt' in data) {
+      return (
+        <VideoConfMessage>
+          <VideoConfMessageRow>
+            <VideoConfMessageContent>
+              <VideoConfMessageIcon />
+              <VideoConfMessageText>{t('Call_ended')}</VideoConfMessageText>
+            </VideoConfMessageContent>
+            <VideoConfMessageActions>
+              <VideoConfMessageAction icon='info' onClick={openCallInfo} />
+            </VideoConfMessageActions>
+          </VideoConfMessageRow>
+          <VideoConfMessageFooter>
+            {data.type === 'direct' && (
+              <>
+                <VideoConfMessageButton onClick={callAgainHandler}>
+                  {t('Call_back')}
+                </VideoConfMessageButton>
+                <VideoConfMessageFooterText>
+                  {t('Call_was_not_answered')}
+                </VideoConfMessageFooterText>
+              </>
+            )}
+            {data.type !== 'direct' &&
+              (data.users.length ? (
+                <>
+                  <VideoConfMessageUserStack>
+                    {data.users.map(({ username }, index) =>
+                      index <= MAX_USERS ? (
+                        <Avatar
+                          size='x28'
+                          key={index}
+                          data-tooltip={username}
+                          url={getUserAvatarPath(username as string)}
+                        />
+                      ) : (
+                        <></>
+                      )
+                    )}
+                  </VideoConfMessageUserStack>
+                  <VideoConfMessageFooterText>
+                    {data.users.length > MAX_USERS
+                      ? `+ ${data.users.length - MAX_USERS} ${t('Joined')}`
+                      : t('Joined')}
```

Comment:
dont concatenate use the translation to do that

Reply:
it was necessary because of the "+" sign, there's a way to render it programmatically using a translation key?

Reply:
Make it part of the translated text

Reply:
yup yup sorry I ate some *** today!